cmd/buildah/push.go

```diff
@@ -179,3 +215,32 @@ func getListOfTransports() string {
 	allTransports := strings.Join(transports.ListNames(), ",")
 	return strings.Replace(allTransports, ",tarball", "", 1)
 }
+
+// getAllTags gets all of the locally tagged images and adds them
+// to the passed in slice.
+func getAllTags(imageName string, imageTags *[]string, store storage.Store) error {
+
+	imageName = "/" + imageName
+	images, err := store.Images()
+	if err != nil {
+		return errors.Wrapf(err, "error reading images")
+	}
+	for _, image := range images {
+
+		if len(image.Names) < 1 {
+			continue
+		}
+		// Name should look like: "docker.io/library/alpine:latest"
+		for _, name := range image.Names {
+			splitName := strings.Split(name, ":")
+			if !strings.HasSuffix(splitName[0], imageName) {
```

Comment:
Is this enough?  What happens if I am looking for image frank, and I have an image named ballparkfrank?

Reply:
I think @mtrmac might have a much better way of doing this.

Reply:
If I put in the image name 'frank' it won't match up to 'ballparkfrank' as I force add a '/' to the imageName up at line 223.  So I think this is OK, but happy to rework it if there's a better way.

Reply:
(BTW `FindImage` just loops over the result of `ResolveNames`. That’s not obviously _correct_ for transport-qualified names, or for storage IDs, but a similar loop here would at least be consistent with that code.)